src/main/java/org/gridsuite/boundary/importer/job/CgmesBoundaryServiceRequester.java

```diff
@@ -92,28 +93,29 @@ public boolean importBoundary(TransferableFile boundaryFile) throws IOException,
         return HttpRequest.BodyPublishers.ofByteArrays(byteArrays);
     }
 
-    public List<String> getBoundariesIds() throws InterruptedException {
+    public List<BoundaryInfo> getBoundariesInfos() throws InterruptedException {
         try {
             HttpRequest request = HttpRequest.newBuilder()
-                .uri(URI.create(serviceUrl + API_VERSION + "/boundaries/ids"))
+                .uri(URI.create(serviceUrl + API_VERSION + "/boundaries/infos"))
                 .GET()
                 .build();
 
             HttpResponse<String> response = httpClient.send(request, HttpResponse.BodyHandlers.ofString());
             if (response.statusCode() == 200) {
                 String json = response.body();
 
-                List<String> result = new ArrayList<>();
+                List<BoundaryInfo> result = new ArrayList<>();
                 JSONArray array = new JSONArray(json);
                 for (int i = 0; i < array.length(); i++) {
-                    result.add(array.getString(i));
+                    var obj = array.getJSONObject(i);
+                    result.add(new BoundaryInfo(obj.getString("id"), obj.getString("filename"), LocalDateTime.parse(obj.getString("scenarioTime"))));
```

Comment:
Why do not user an object mapper ?
            if (response.statusCode() == 200) {
                ObjectMapper mapper = new ObjectMapper();
                mapper.registerModule(new JavaTimeModule());
                return mapper.readValue(response.body(), new TypeReference<List<BoundaryInfo>>(){});
            } else {
                LOGGER.error(response.toString());
            }

Reply:
Done